src/libsystemd/sd-bus/bus-objects.c

```diff
@@ -1076,10 +1076,6 @@ static int object_manager_serialize_path(
                         if (r < 0)
                                 return r;
 
-                        r = sd_bus_message_append(reply, "{sa{sv}}", "org.freedesktop.DBus.ObjectManager", 0);
-                        if (r < 0)
-                                return r;
-
```

Comment:
dropping this just like that sounds a bit too simplistic though. We should drop this only on objects where the object manager is not turned on. For those where it is, we should really return the info, i am sure

Reply:
Is nesting ObjectManagers even supported?

Should the outer ObjectManager tracks all object of children ObjectManagers?